certified-connectors/Tikit/readme.md

```diff
@@ -0,0 +1,7 @@
+## Tikit
+
+Tikit for Microsoft Teams enables conversational ticketing in a system you already use. Turn messages from chat into tickets, authenticate with M365 single sign on, deflect common questions with artificial intelligence and report with Power BI.
+
+## Prerequisites
+
+You will need to have an active Tikit, [Microsoft 365 Service Desk](https://www.tikit.ai/pricing/) subscription to leverage this connector.
```

Comment:
@TikitTeam Please refer the template provided for certified-connectors and update the readme.md file accordingly. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
updated